book/07-git-tools/sections/submodules.asc

```diff
@@ -384,199 +384,199 @@ Ito ay nangangahulugan na walang lokal na tinatrabahuang branch (tulad ng ``mast
 Ng walang tinatrabahuang branch na nagsusubaybay ng mga pagbabago, iyon ay nangangahulugan na kahit na ikaw ay mag-commit ng mga pagbabago sa submodule, ang mga pagbabagong iyon ay posibleng mawala sa susunod na patakbuhin mo ang `git submodule update`.
 Kailangan mong gawin ang ilang mga dagdag na hakbang kung nais mong baguhin ang mga submodule na susubaybayan.
 
-In order to set up your submodule to be easier to go in and hack on, you need do two things.
-You need to go into each submodule and check out a branch to work on.
-Then you need to tell Git what to do if you have made changes and then `git submodule update --remote` pulls in new work from upstream.
-The options are that you can merge them into your local work, or you can try to rebase your local work on top of the new changes.
+Para ma-set up ang iyong submodule na maging mas madaling pasukin at i-hack, kailangan mong gawin ang dalawang bagay.
+Kailangan mong puntahan ang bawat submodule at suriin ang isang branch na tatrabahuin.
+Pagkatapos ay kailangan mong sabihan ang Git kung ano ang gagawin kung may ginawa kang mga pagbabago at pagkatapos ay ipu-pull ng `git submodule update --remote` sa bagong trabaho mula sa upstream.
+Ang mga pagpipilian ay maaari kang mag-merge ng mg ito sa iyong lokal na trabaho, o subukang i-rebase ang iyong lokal na trabaho sa itaas ng mga bagong pagbabago.
 
-First of all, let's go into our submodule directory and check out a branch.
+Una sa lahat, puntahan natin ang ating submodule na direktoryo at suriin ang isang branch.
 
 [source,console]
 ----
 $ git checkout stable
-Switched to branch 'stable'
+Pinalitan sa branch na 'stable'
 ----
 
-Let's try it with the ``merge'' option.
-To specify it manually, we can just add the `--merge` option to our `update` call.
-Here we'll see that there was a change on the server for this submodule and it gets merged in.
+Subukan natin ito sa opsyon na ``merge''.
+Upang itakda ito nang manu-mano, maaari lamang nating idagdag ang pagpipilian na `--merge` sa ating `update` na pagtawag.
+Dito, makikita natin na may isang pagbabago sa server para sa submodule na ito at ito ay nai-merge.
```

Comment:
Please, change "manu-mano" to "mano-mano" base on Google Translate and Glosbe it was spelled like that.